server/embed/config_test.go

```diff
@@ -97,6 +97,7 @@ func TestConfigFileFeatureGates(t *testing.T) {
 		name                                string
 		serverFeatureGatesJSON              string
 		experimentalStopGRPCServiceOnDefrag string
+		experimentalInitialCorruptCheck     string
```

Comment:
Please also add a test case to verify both legacy flag and feature gate can't be set both, similar to 

https://github.com/etcd-io/etcd/blob/5485c710674b5990f62e83984d180b4b552fa52c/server/embed/config_test.go#L110-L115

Reply:
done